source/release-notes/1.4.txt

```diff
@@ -27,7 +25,54 @@ Starting in 1.4.0, you can set the workload level that
 destination clusters. For more information, see the
 :option:`--loadLevel` option.
 
+Improved Resiliency
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` 1.4.0 improves :term:`oplog` handling. The ``oplog``
+handling improvement adds resiliency to migrations and reduces the time
+to complete large migrations.
```

Comment:
I find this a little confusing. The thing we're now more resilient is specifically "oplog rollover". The problem with older versions is that if the first part of a migration took (say) 48 hours, and we only had 24 hours of oplog entries available, mongosync could not complete the migration. Now, we do more work in the first part of the migration so that there's less to do in the second part.

I'm not sure how best to describe that, but I'm tagging @edobranov here, who might have additional thoughts.